src/lib/definitions.js

```diff
@@ -22,31 +22,38 @@ class TSMKeyword {
 }
 
 export const symbols = [
-  new TSMKeyword('DBHistorical', 'Historical Price'),
-  new TSMKeyword('DBMarket', 'Market Value'),
-  new TSMKeyword('DBMinBuyout', 'Minimum Buyout Value'),
-  new TSMKeyword('DBRegionHistorical', 'Region Historical Price'),
-  new TSMKeyword('DBRegionMarketAvg', 'Region Market Value Average'),
-  new TSMKeyword('DBRegionMinBuyoutAvg', 'Region Minimum Buyout Average'),
-  new TSMKeyword('DBRegionSaleAvg', 'Region Sale Average'),
-  new TSMKeyword('DBRegionSaleRate', 'Number of sales of the item compared to how many expire or get cancelled'),
-  new TSMKeyword('DBRegionSoldPerDay', 'Number of times the item sells per day on average'),
-  new TSMKeyword('NumExpires', 'Number of times the item has expired since the last time it was sold'),
-  new TSMKeyword('ItemQuality', 'Item Quality (0 = poor, 1 = common, 2 = uncommon, 3 = rare, 4 = epic, 5 = legendary)'),
-  new TSMKeyword('ItemLevel', 'The item level of the item'),
-  new TSMKeyword('RequiredLevel', 'The level required to use or equip the item'),
-  new TSMKeyword('Destroy', 'Value of materials if you were to destroy the item'),
-  new TSMKeyword('VendorBuy', 'Cost if you were to buy the item from a vendor'),
-  new TSMKeyword('VendorSell', 'Value if you were to sell the item to the vendor'),
-  new TSMKeyword('AvgBuy', 'Average Buy Price'),
-  new TSMKeyword('AvgSell', 'Average Sell Price'),
-  new TSMKeyword('Crafting', 'Total cost of crafting the item (sum of all MatPrices for the item)'),
-  new TSMKeyword('MatPrice', 'The cost of a single crafting material'),
-  new TSMKeyword('MinSell', 'Minimum sell price that you sold the item'),
-  new TSMKeyword('MinBuy', 'Minimum buy price that you bought the item'),
-  new TSMKeyword('MaxBuy', 'Max buy price that you purchased the item'),
-  new TSMKeyword('MaxSell', 'Max sell price that you sold the item'),
-  new TSMKeyword('BaseItem', 'Reference the base item (without item variations)'),
+  new TSMKeyword('SmartAvgBuy', 'Returns the average purchase price across your last purchases of the item'),
+  new TSMKeyword('AuctioningOpMax', 'Returns the maximum price generated by the operations applied to your groups'),
+  new TSMKeyword('AuctioningOpMin', 'Returns the minimum price generated by the operations applied to your groups'),
+  new TSMKeyword('AuctioningOpNormal', 'Returns the normal price generated by the operations applied to your groups'),
+  new TSMKeyword('AvgBuy', 'Returns the average buy price for all purchases for a particular item'),
+  new TSMKeyword('AvgSell', 'Returns the average sell price for all sales for a particular item'),
+  new TSMKeyword('Crafting', 'Returns the sum of the material prices of each individual item used to craft the recipe'),
+  new TSMKeyword('DBHistorical', 'Returns the 60-day average of the DBMarket value for a particular item on your realms Auction House'),
```

Comment:
Could you change "realms" to read "realm\'s" to stay consistent with the other changes you made?

Reply:
Done